src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(health, name) {
+    this.health = 100;
+    this.name = name;
+    Animal.alive.push(this);
+  }
+
+  removeDead() {
+    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health) {
```

Comment:
[CONSTRUCTOR PARAMETERS] - The `Herbivore` class should not accept a `health` parameter in its constructor as well since it extends `Animal` and the health is always initialized to `100`. Please remove the `health` parameter from the constructor signature.